api/v1/.htaccess

```diff
@@ -7,6 +7,8 @@ php_value include_path ".:../..:/var/www/common:/usr/share/pear"
 
 Header Unset Expires
 SetEnvIf Origin ^(https?://.+\.burningflipside.com(?::\d{1,5})?)$   CORS_ALLOW_ORIGIN=$1
+SetEnvIf Origin ^(https?://localhost(?::\d{1,5})?)$   CORS_ALLOW_ORIGIN=$1
```

Comment:
We actually don't want to let this happen in production... (This can result in cross site scripting type attacks) We should figure out a better way to dynamically set CORS headers.